salt/metalk8s/addons/nginx-ingress-control-plane/certs/server.sls

```diff
@@ -31,6 +30,7 @@ Create Control-Plane Ingress server private key:
     'nginx-ingress-control-plane.metalk8s-ingress.svc',
     'nginx-ingress-control-plane.metalk8s-ingress.svc.cluster.local',
     grains.metalk8s.control_plane_ip,
```

Comment:
Should we keep the local IP, since it was there only to represent the Service IP?

Reply:
It's not needed indeed, I will remove it